app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -70,7 +70,7 @@ def connect_identity
 
   def event_monitor_options
     @event_monitor_options ||= begin
-      opts = {:ems => self}
+      opts = {:ems => self, :automatic_recovery => false, :recover_from_connection_close => false}
```

Comment:
yes, byt default it's all of, so we don;t have any infinite loops anywhere, like validation call.

Then just for parameters, that are used for starting actual event catcher, we are adding params reconnect, so the worker can gracefully handle amqp restarts on the other side and other connection breaking events.